src/connectors/aws_collect.py

```diff
@@ -459,6 +459,14 @@
         ('target_grants', 'VARIANT'),
         ('target_prefix', 'STRING'),
     ],
+    # https://docs.aws.amazon.com/cli/latest/reference/s3api/get-bucket-tagging.html#output
+    's3_get_bucket_tagging': [
+        ('recorded_at', 'TIMESTAMP_LTZ'),
+        ('account_id', 'STRING'),
+        ('error', 'VARIANT'),
+        ('bucket', 'STRING'),
```

Comment:
I think I keep messing up the order of these

Reply:
although error then bucket seems to be the more common one

Reply:
all good, it's an unfortunate limitation. the params go before error, and then either "error" or everything after it is populated with response. if all of them are NULL then there was no error and the response was empty.

Reply:
bucket should be after error if the bucket is part of the response and before the error if it's part of the query

Reply:
thank you